### ISSUE TITLE
activated autocomplete-bibtex package does not save session

### ISSUE THREAD

**Issue by mbroedl** [apcshields/autocomplete-bibtex#98]
Sorry that it's me again.

Just noticed the other day, that my sessions were not saved anymore, but atom always went back to a session from a few days ago.
Activated and deactivated some of the most recently updated packages, and it turned out that autocomplete-bibtex is the evil guy. :(

So: when the package is activated, a changed session is not saved (or recalled?) by atom. Deactivating the package before closing atom does not help either.

I've no idea what might cause it.

Atom 1.23.3
autocomplete-bibtex 1.2.0

**Comment by mangecoeur** [comment]
Bleh, that's wierd. What do you mean by session exactly? Personally I have no issues getting back my last opened files on windows and mac...

Maybe you could try this for me: open the developer console and then close atom. Tell me if you see a flash of red error messages on the console output just before the windows closes. This might be indicative of autocomplete causing an error when it saves its current state.

**Comment by mbroedl** [comment]
With session I mean that it stores all open documents and the current status of the documents regardless of whether they were saved. So when you close atom, and then open atom again, you have the same files with the same saved or unsaved file-state open.
With autocomplete-bibtex, it stops recording these changes once I activate the package.

I get this error message when activating autocomplete-bibtex (and every time I make a change to any of the documents, or on opening/closing them).
![image](https://user-images.githubusercontent.com/19365355/36784589-dfddc126-1c77-11e8-9973-bbccfe675f5e.png)
I then had to take a screencast to actually get to the error message you hypothesised, but maybe it's not related to autocomplete-bibtex, but only the former one?
![image](https://user-images.githubusercontent.com/19365355/36784621-fb1cb6cc-1c77-11e8-8b84-b1ac1ebfcc05.png)

**Comment by mangecoeur** [comment]
Ok I think i know what this is and how to fix it - I thought I had disabled saving the state of autocomplete bibtex (which I did because I didn't finish work to make it work properly with the auto-updating of the reference files). But it looks like it is trying to save it still. As a temporary fix, try commenting out lines 39-49 in provider.js . I will try a proper fix when I have time.

**Comment by mbroedl** [comment]
Yes, this works perfectly! No more error messages, thanks a lot. :)
